cirq-core/cirq/ops/common_channels.py

```diff
@@ -178,10 +179,12 @@ def p_z(self) -> float:
             raise ValueError('num_qubits should be 1')
         return self._error_probabilities.get('Z', 0.0)
 
-    @property
-    def num_qubits(self) -> int:
-        """The number of qubits"""
-        return self._num_qubits
+    # The following property was deprecated in Cirq v0.14. Please use the num_qubits methos instead.
+    #
+    # @property
+    # def num_qubits(self) -> int:
+    #     """"The number of qubits"""
+    #     return self._num_qubits
```

Comment:
Optional: I'd remove the commented out code and just keep the comment (which you can also move to the top of the class). Folks affected by this will likely search for "num_qubits" so as long as the comment contains that string this remains discoverable.

Reply:
done